docs/contributing-to-airbyte/tutorials/building-a-python-destination.md

```diff
@@ -0,0 +1,193 @@
+# Building a Python Destination
+
+## Summary
+
+This article provides a checklist for how to create a Python destination. Each step in the checklist has a link to a more detailed explanation below.
+
+## Requirements
+
+Docker and Python with the versions listed in the [tech stack section](../../understanding-airbyte/tech-stack.md). You can use any Python version between 3.7 and 3.9, but this tutorial was tested with 3.7.  
+
+## Checklist
+
+### Creating a destination
+
+* Step 1: Create the destination using the template generator
+* Step 2: Setup the virtual environment 
+* Step 3: Implement `spec` to define the configuration required to run the connector
+* Step 4: Implement `check` to provide a way to validate configurations provided to the connector
+* Step 5: Implement `write` to write data to the destination
+* Step 6: Set up Acceptance Tests
```

Comment:
since this is not available for Python destinations, should we remove this for now?

Reply:
This is WIP in the next day or two so IMO we should keep it (it also makes it clear this is coming)